Drop SysV and Upstart support

jooola opened the pull request:
Closes #1247

I didn't change much in the installer script as I don't know yet whether we should clean it up or rewrite it.

Mostly remove sysv files and cleanup the setup.py files.

This can be reviewed, but I feel I am missing something, hence the WIP state.

paddatrapper commented on python_apps/pypo/setup.py:
```diff
@@ -41,9 +13,18 @@
     url="http://github.com/sourcefabric/Airtime",
     author="sourcefabric",
```
And the URL/author here?

paddatrapper reviewed (commented):
I don't see anywhere else these are used

jooola commented:
Removed sysv, upstart, updated all the setup.py files with recent project details.

Important change, I relicenced the airtime-celery service to AGPLv3, are we OK with that ? I don't know if this is legally ok ?

Otherwise, should be good to go, we will still be able to cleanup files later on.

jooola commented:
I'll open another PR to update the setup.py details.
